site/tracesdb-engine/index.html

```
@@ -0,0 +1,255 @@
+      <!-- ─── Section 1: Dataset ──────────────────────────────── -->
+      <section class="panel" id="section-dataset">
+        <h2>Load a Dataset</h2>
+        <p class="section-intro">
+          Choose a trace scenario to generate. Each creates realistic distributed traces
+          with multiple services, varying depths, and configurable error rates.
+        </p>
+
+        <div class="scenario-grid" id="scenarioGrid"></div>
+
+        <div class="controls-row">
+          <button class="btn btn-primary" id="btnGenerate" disabled>Generate Traces</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)